### PR TITLE
Added ASP.Net.Core sample in ASP.NET Web Forms

### PR DESCRIPTION
**Pull request by snehasf3509**
Added ASP.Net.Core sample in ASP.NET Web Forms

### CODE REVIEW

File: Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/MainPage.aspx.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            using (WordDocument document = new WordDocument())
+            {
+                 //Adds new section to the document.
+                IWSection section = document.AddSection();
+                //Adds new paragraph to the section.
+                IWParagraph paragraph = section.AddParagraph();
+                //Appends the text to the created paragraph.
+                paragraph.AppendText("Adventure Works Cycles, the fictitious company on which the AdventureWorks sample databases are based, is a large, multinational manufacturing company.");
+
+				#region Save and download document
```
Comment (MohanaselvamJothi) [inline Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/MainPage.aspx.cs:32]: clear spacing

File: Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/Send-Word-to-client-browser.csproj.user [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (MohanaselvamJothi) [inline Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/Send-Word-to-client-browser.csproj.user:1]: Delete csproj.user

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/MainPage.aspx.cs (hunk 1 above)
Comment (AkashArul26) [reply Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/MainPage.aspx.cs]: Addressed the feedback

File: Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/Send-Word-to-client-browser.csproj.user (hunk 2 above)
Comment (AkashArul26) [reply Read-and-Save-document/Send-Word-to-client-browser/ASP.NET/Send-Word-to-client-browser/Send-Word-to-client-browser.csproj.user]: Addressed the feedback

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**